releases/release-7.3.0.md

```diff
@@ -0,0 +1,352 @@
+---
+title: TiDB 7.3.0 Release Notes
+summary: Learn about the new features, compatibility changes, improvements, and bug fixes in TiDB 7.3.0.
+---
+
+# TiDB 7.3.0 Release Notes
+
+Release date: xx xx, 2023
+
+TiDB version: 7.3.0
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.3/quick-start-with-tidb) | [Installation packages](https://www.pingcap.com/download/?version=v7.3.0#version-list)
+
+7.3.0 introduces the following major features. The rest of the release (detailed in the Details section) was a series of enhancements to query stability in TiDB server and TiFlash. These are more miscellaneous in nature and not user-facing so they are not included in this section dedicated to release highlights:
+
+<table>
+<thead>
+  <tr>
+    <th>Category</th>
+    <th>Feature</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>Scalability and Performance</td>
+    <td>TiDB Lightning supports Partitioned Raft KV </td>
+    <td>TiDB Lightning now supports the new Partitioned Raft KV architecture, as part of the near-term GA of the architecture.
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="2">Reliability and Availability</td>
+    <td><a href="https://docs.pingcap.com/tidb/v7.3/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#conflict-detection">Add automatic conflict detection and resolution on imports</a></td>
+    <td>The TiDB Lightning Physical Import Mode supports the new version of conflict detection and implementing the semantics of replacing (`replace`) or ignoring (`ignore`) conflicting data when encountering conflicts. It automatically handles conflicting data for you while improving the performance of conflict resolution.</td>
+  </tr>
+  <tr>
+    <td><a href="https://docs.pingcap.com/tidb/v7.3/tidb-resource-control.md#query-watch-parameters">Manual management of runaway queries </a>(experimental)</td>
+    <td>Query timeouts exist on a per-TiKV node basis but now Resource Groups can manage queries by global parallel execution time and either deprioritize or kill them. Allowing operators to mark target queries by SQL manually Text, SQL Digest or Plan Digest and what do with the queries at a Resource Group level, gives them much more control over the impact unexpected large queries may have on the cluster.</td>
+  </tr>
```

Comment:
What does "global parallel execution time" mean? It is not mentioned in https://docs.pingcap.com/tidb/dev/tidb-resource-control#query-watch-parameters.

Reply:
It's from criteria of runaway management. I updated the description and make it more focused on manual management part. PTAL.

Reply:
LGTM